src/Microsoft.DotNet.XHarness.Apple/Orchestration/BaseOrchestrator.cs

```diff
@@ -85,6 +85,11 @@ public abstract class BaseOrchestrator : IDisposable
                 }
             }
 
+            if (includeWirelessDevices && target.Platform.IsSimulator())
+            {
+                _logger.LogWarning("Including wireless devices while targeting a simulator has no effect");
```

Comment:
It takes no effect but doesn't it increase timeout up to 20 seconds in that case? I mean if yes probably you should force its value to false

Reply:
@greenEkatherine valid observation but for simulators we do an entirely different mlaunch call and it has different timeouts